proposed/http-handlers/request-handlers.md

```diff
@@ -110,20 +110,20 @@ use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 /**
+ * Participant in processing a server request and response
+ *
  * An HTTP middleware component participates in processing an HTTP message,
- * either by acting on the request or the response. This interface defines the
- * methods required to use the middleware.
+ * either by acting on the request, generating the response, or forwarding the
+ * request to a subsequent middleware and possibly acting on its response.
  */
 interface MiddlewareInterface
 {
     /**
-     * Process an incoming server request and return a response, optionally delegating
-     * response creation to a handler.
-     *
-     * @param ServerRequestInterface $request
-     * @param RequestHandlerInterface $handler
+     * Process an incoming server request
```

Comment:
Period please. :smile:

Reply:
> Processes a server request and produces a response

? That makes it consistent with the docblock of the `HandlerInterface`, the only difference being handles/processes.